### PR TITLE
fix unstable stress test

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@Andrewmatilde PTAL

**Comment by Andrewmatilde** [comment]
I am confused about how this problem works. Do we need new a issue or add some message about it?

**Comment by WangXiangUSTC** [comment]
> I am confused about how this problem works. Do we need new a issue or add some message about it?

https://github.com/chaos-mesh/chaosd/blob/6cedaf5f191d8fc25eec95c16309c17324b9c55d/pkg/server/chaosd/stress_attack.go#L119

when recover stress chaos, chaosd will kill the stress-ng process. But `kill` will only send a signal to stress-ng, will not wait for the process exit. Maybe stress-ng exit cost more than 1 second, and then the test failed.

And I print the stress-ng's process info when test failed @Andrewmatilde

File: test/integration_test/stress/run.sh [hunk 1]
```diff
@@ -32,11 +32,11 @@ fi
 uid=`cat cpu.out | grep "Attack stress cpu successfully" | awk -F: '{print $2}'`
 ${bin_path}/chaosd recover ${uid}
 
-sleep 1
+sleep 2
```
Comment (Andrewmatilde) [inline test/integration_test/stress/run.sh:35]: How about wait for process here?

Comment (cwen0) [reply test/integration_test/stress/run.sh:35]: Agree with this

Comment (WangXiangUSTC) [reply test/integration_test/stress/run.sh]: good suggestion, fixed, PTAL again

Comment (Andrewmatilde) [reply test/integration_test/stress/run.sh]: LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**